cmd/wfx/cmd/config/flags.go

```diff
@@ -74,6 +77,8 @@ func NewFlagset() *pflag.FlagSet {
 	f.StringSlice(SchemeFlag, []string{"http"}, "the listeners to enable, this can be repeated and defaults to the schemes in the swagger spec")
 	f.Duration(CleanupTimeoutFlag, 10*time.Second, "grace period for which to wait before killing idle connections")
 	f.Duration(GracefulTimeoutFlag, 15*time.Second, "grace period for which to wait before shutting down the server")
+	f.Duration(PingIntervalSSEFlag, DefaultPingIntervalSSE, "interval to send periodic keep-alive messages to prevent the connection from being closed")
```

Comment:
Shouldn't this explicitly name *which* connection is prevented from being closed? I.e., explicitly state that this is for covering SSE for job notifications.

Reply:
Clarified (although the name of the flag contains the word `sse` ;)